### PR TITLE
Nftlist component with mock json

### PR DESCRIPTION
**Pull request by bhumitaylor14**
https://linear.app/pillarproject/issue/PRO-2128/etherspot-ui-nft-list

## Description

Create NFT list component

## How Has This Been Tested?

- tested on local

## Screenshots (if appropriate):
Screenshot 2024-03-04 at 11 42 09 AM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by bhumitaylor14** [comment]
@kio @lalitkumarrock As per discussion over standup call, I am creating list using mockdata of nftlist. Once project key issue will solve I will uncomment function  and remove mock logic.

**Review by lalitkumarrock — APPROVED** [review]
LGTM!

File: src/data/mock_nft.json
Comment (lalitkumarrock) [inline src/data/mock_nft.json:1]: Remove this file if not required.

File: src/models/NftList.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  visibleNoDataText?: boolean;
+  unorderedListContainerClassName?: string;
+  listItemContainerClassName?: string;
+  unorderedListIterableContainerClassName?: string;
+  nftMediaClassName?: string;
+  nftImageClassName?: string;
+  nftNameClassName?: string;
+  nftTokenIdClassName?: string;
+  nftTokenAmountClassName?: string;
+  dataApiKey?: string;
```
Comment (lalitkumarrock) [inline src/models/NftList.ts]: `dataApiKey` Move it to Common.ts if used in multiple components

**Review by lalitkumarrock — CHANGES_REQUESTED** [review]

**Review by lalitkumarrock — APPROVED** [review]

**Pull request merged**